app/services/food_service.rb

```diff
@@ -0,0 +1,15 @@
+class FoodService
+ 
+    def self.search_food(food)
+        response = conn.get("search?query=#{food}")
+        JSON.parse(response.body, symbolize_names: true) 
+      end
+
+    private  
+
+    def self.conn
+        Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/foods/") do |faraday|
```

Comment:
Typically, the Faraday connection would set up the connection with *just* the base URL, so everything up until .gov. The `/fdc/v1/foods/search` part would be in the path on line 4. In this setup, you have part of the path here and part on line 4, which is just a bit less conventional and can cause problems